modules/integration/src/test/java/io/fluo/integration/impl/WeakNotificationOverlapIT.java

```diff
@@ -128,7 +127,7 @@ public void testOverlap() throws Exception {
     ttx7.mutate().row(1).col(ntfyCol).weaklyNotify();
     ttx7.done();
 
-    Assert.assertEquals(2, countNotifications());
+    Assert.assertEquals(1, countNotifications());
 
     new TotalObserver().process(ttx6, Bytes.of("1"), ntfyCol);
     // should not delete notification created by ttx7
```

Comment:
I am not familiar with this test.  However, given that the count has changed, are the comments still accurate?